bokehjs/src/lib/models/annotations/box_annotation.ts

```diff
@@ -123,11 +125,13 @@ export namespace BoxAnnotation {
     ew_cursor: p.Property<string | null>
     ns_cursor: p.Property<string | null>
     in_cursor: p.Property<string | null>
+    /** @deprecated */
+    render_mode: p.Property<RenderMode>
```

Comment:
deprecation sholud go in release notes

Reply:
This was deprecated way before, just not indicated in bokehjs.